tests/testdata/02-simple.c

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
These source files are the ones used when building the samples, they are 1:1 matches from the ones generated in the .sh script.